setup.py

```diff
@@ -31,9 +31,8 @@
         package_dir={"fpdf": "fpdf"},
         install_requires=[
             "defusedxml",
-            "svg.path",
             "Pillow>=6.2.2",  # minimum version tested there: https://github.com/PyFPDF/fpdf2/actions/runs/2295868575
-            "fonttools",
+            "fonttools>=4.37.2",
```

Comment:
The current minium version is `4.30` on `master`: is this really the minimum version now @gmischler?

Reply:
The absolute minimum is [4.34.0](https://github.com/fonttools/fonttools/releases/tag/4.34.0), which is when the exponentials were fixed. I just picked the most current one at the time, but I don't really care as long as it works.

Reply:
OK then could you change this to 4.34.0 please?